getting_started.ipynb

```diff
@@ -0,0 +1,1962 @@
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "prompt_value = chat_prompt.format_messages(country=\"Japanese\", question=\"What is the most popular Sashimi in Japan vs the rest of the world?\")\n",
+    "\n",
+    "response = model.invoke(prompt_value)\n",
+    "response.content"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 60,
+   "id": "3893edc8-acc0-418e-ac5d-31a65b00b8b8",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "In Japan, the most popular sashimi is often Maguro (tuna), specifically the fatty part known as Otoro. Globally, Salmon sashimi tends to be more popular due to its rich flavor and widespread availability.\n"
+     ]
+    }
+   ],
+   "source": [
+    "from langchain_core.output_parsers import StrOutputParser\n",
+    "\n",
+    "chain = chat_prompt | model | StrOutputParser()\n",
+    "\n",
+    "print(\n",
+    "    chain.invoke(\n",
+    "        {\n",
+    "            \"country\": \"Japanese\", \n",
+    "            \"question\": \"What is the most popular Sashimi in Japan vs the rest of the world?\"\n",
+    "        }\n",
+    "    )\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3f186179-13e7-4056-9b62-9b4b1ac8de60",
+   "metadata": {},
+   "source": [
+    "### Wait in between API calls"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9f747f98-7b78-436f-a889-e3cff3f19bf9",
+   "metadata": {},
+   "source": [
+    "Depending on the users Azure plan and account will determine how many API requests can be made within a given interval. If too many requests are sent frequently an error may occur where the user will be told to wait **x** amount of time before sending another request.\n",
+    "\n",
+    "When creating a model, one of the parameters briefly mentioned earlier was the `max_retries` parameter. When calling the underlying Python OpenAI library, the library will wait and retry the call on your behalf at least 2 times (the default) before raising a RateLimiteError, or whatever `max retries` is set as.\n",
+    "\n",
+    "Otherwise, to ensure the user is not overwhelming the Azure services, make sure to wait between API calls when possible - this can be enforced by using the `time` package."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 65,
+   "id": "99e45cb7-6e35-4420-bd41-651157e5596f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import time\n",
+    "time.sleep(5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c4a85fe6-ee18-4fc8-88c2-b67297ac8ef8",
+   "metadata": {},
+   "source": [
+    "### Check the costs and token usage of a given model API call"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 67,
+   "id": "561dfbfb-0933-4650-8230-23f1bc2b1f46",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from langchain_community.callbacks.manager import get_openai_callback"
```

Comment:
Do we get a deprecation warning with this import? I see this in the code: https://github.com/langchain-ai/langchain/blob/master/libs/langchain/langchain/callbacks/manager.py#L48

I think importing from `from langchain_community.callbacks import get_openai_callback` might be the recommend way, but I haven't tried it.

Reply:
It doesn't give me a deprecation warning but I've changed it to `from langchain_community.callbacks import get_openai_callback` regardless.